src/controller/article.py

```
@@ -71,9 +71,8 @@ def __init__(
         self.session = session
 
     async def create_article(
-        self, request: Request, body: BodyCreateArticle
+        self, body: BodyCreateArticle, current_user: User, request: Request
```

Ruff (0.14.4)

Unused method argument: `request`

(ARG002)